srcpkgs/wluma/template

```diff
@@ -0,0 +1,18 @@
+# Template file for 'wluma'
+pkgname=wluma
+version=1.2.2
+revision=1
+build_style=gnu-makefile
+make_build_target="build"
+hostmakedepends="pkgconf meson"
```

Comment:
You need `wayland-devel` here as well

Reply:
Why?

Reply:
To fix cross-build, because that's the package `wayland-scanner` is in

Reply:
OK